<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Submission</class>
 <widget class="QWidget" name="Submission">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion des Soumissions</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>10</number>
   </property>
   <property name="leftMargin">
    <number>15</number>
   </property>
   <property name="topMargin">
    <number>15</number>
   </property>
   <property name="rightMargin">
    <number>15</number>
   </property>
   <property name="bottomMargin">
    <number>15</number>
   </property>
   <item>
    <widget class="QLabel" name="titleLabel">
     <property name="text">
      <string>Gestion des Soumissions</string>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #2c3e50; font-size: 16pt; font-weight: bold; margin-bottom: 10px;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="styleSheet">
      <string notr="true">QTabWidget::pane {
    border: 1px solid #bdc3c7;
    border-radius: 5px;
    background: white;
}

QTabBar::tab {
    background: rgba(189, 195, 199, 0.3);
    color: #2c3e50;
    padding: 10px 30px;
    margin-right: 2px;
    border-top-left-radius: 5px;
    border-top-right-radius: 5px;
    font-weight: bold;
}

QTabBar::tab:selected {
    background: rgba(52, 152, 219, 0.8);
    color: white;
}

QTabBar::tab:hover {
    background: rgba(52, 152, 219, 0.5);
}</string>
     </property>
     <widget class="QWidget" name="tabForm">
      <attribute name="title">
       <string>Créer une Soumission</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QGroupBox" name="formGroup">
         <property name="title">
          <string>Détails de la Soumission</string>
         </property>
         <property name="styleSheet">
          <string notr="true">QGroupBox { font-weight: bold; border: 2px solid #bdc3c7; border-radius: 5px; margin-top: 10px; padding-top: 10px; }
QGroupBox::title { subcontrol-origin: margin; left: 10px; padding: 0 5px; }</string>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <property name="horizontalSpacing">
           <number>10</number>
          </property>
          <property name="verticalSpacing">
           <number>15</number>
          </property>
          <property name="leftMargin">
           <number>20</number>
          </property>
          <property name="topMargin">
           <number>20</number>
          </property>
          <property name="rightMargin">
           <number>20</number>
          </property>
          <property name="bottomMargin">
           <number>20</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="label_titre">
            <property name="text">
             <string>Titre:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="titleEdit">
            <property name="placeholderText">
             <string>Titre de l'article</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_auteur">
            <property name="text">
             <string>Auteur:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="authorEdit">
            <property name="placeholderText">
             <string>Nom de l'auteur</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_domaine">
            <property name="text">
             <string>Domaine:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="domainEdit">
            <property name="placeholderText">
             <string>Domaine de recherche</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_date">
            <property name="text">
             <string>Date:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QDateEdit" name="dateEdit">
            <property name="readOnly">
             <bool>true</bool>
            </property>
            <property name="calendarPopup">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_statut">
            <property name="text">
             <string>Statut:</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QComboBox" name="statusCombo">
            <item>
             <property name="text">
              <string>En attente</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>En révision</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Accepté</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Rejeté</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_relecteur">
            <property name="text">
             <string>Relecteur:</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QLineEdit" name="reviewerEdit">
            <property name="placeholderText">
             <string>Nom du relecteur</string>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="label_resume">
            <property name="text">
             <string>Résumé:</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QTextEdit" name="abstractEdit">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>120</height>
             </size>
            </property>
            <property name="placeholderText">
             <string>Résumé de l'article...</string>
            </property>
           </widget>
          </item>
          <item row="7" column="0" colspan="2">
           <layout class="QHBoxLayout" name="crudLayout">
            <property name="spacing">
             <number>10</number>
            </property>
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="addBtn">
              <property name="minimumSize">
               <size>
                <width>120</width>
                <height>40</height>
               </size>
              </property>
              <property name="text">
               <string>Ajouter</string>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton { background-color: rgba(46, 204, 113, 0.7); color: white; padding: 8px 20px; border-radius: 4px; font-weight: bold; font-size: 11pt; }
QPushButton:hover { background-color: rgba(46, 204, 113, 1.0); }</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="updateBtn">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="minimumSize">
               <size>
                <width>120</width>
                <height>40</height>
               </size>
              </property>
              <property name="text">
               <string>Modifier</string>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton { background-color: rgba(243, 156, 18, 0.7); color: white; padding: 8px 20px; border-radius: 4px; font-weight: bold; font-size: 11pt; }
QPushButton:hover { background-color: rgba(243, 156, 18, 1.0); }
QPushButton:disabled { background-color: rgba(243, 156, 18, 0.3); }</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="deleteBtn">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="minimumSize">
               <size>
                <width>120</width>
                <height>40</height>
               </size>
              </property>
              <property name="text">
               <string>Archiver</string>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton { background-color: rgba(231, 76, 60, 0.7); color: white; padding: 8px 20px; border-radius: 4px; font-weight: bold; font-size: 11pt; }
QPushButton:hover { background-color: rgba(231, 76, 60, 1.0); }
QPushButton:disabled { background-color: rgba(231, 76, 60, 0.3); }</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="clearBtn">
              <property name="minimumSize">
               <size>
                <width>120</width>
                <height>40</height>
               </size>
              </property>
              <property name="text">
               <string>Effacer</string>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton { background-color: rgba(149, 165, 166, 0.7); color: white; padding: 8px 20px; border-radius: 4px; font-weight: bold; font-size: 11pt; }
QPushButton:hover { background-color: rgba(149, 165, 166, 1.0); }</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabList">
      <attribute name="title">
       <string>Liste des Soumissions</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="toolbarLayout">
         <property name="spacing">
          <number>10</number>
         </property>
         <item>
          <widget class="QLabel" name="searchLabel">
           <property name="text">
            <string>Rechercher:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="searchEdit">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>0</height>
            </size>
           </property>
           <property name="placeholderText">
            <string>Entrez votre recherche...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="searchTypeCombo">
           <item>
            <property name="text">
             <string>Par Titre</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Par Domaine</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="searchBtn">
           <property name="text">
            <string>Rechercher</string>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton { background-color: rgba(52, 152, 219, 0.7); color: white; padding: 5px 15px; border-radius: 3px; }
QPushButton:hover { background-color: rgba(52, 152, 219, 1.0); }</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="sortLabel">
           <property name="text">
            <string>Trier:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="sortTypeCombo">
           <item>
            <property name="text">
             <string>Par Date</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Par Titre</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="sortBtn">
           <property name="text">
            <string>Trier</string>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton { background-color: rgba(155, 89, 182, 0.7); color: white; padding: 5px 15px; border-radius: 3px; }
QPushButton:hover { background-color: rgba(155, 89, 182, 1.0); }</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="exportBtn">
           <property name="text">
            <string>Exporter PDF</string>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton { background-color: rgba(39, 174, 96, 0.7); color: white; padding: 5px 15px; border-radius: 3px; }
QPushButton:hover { background-color: rgba(39, 174, 96, 1.0); }</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="statsBtn">
           <property name="text">
            <string>Statistiques</string>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton { background-color: rgba(230, 126, 34, 0.7); color: white; padding: 5px 15px; border-radius: 3px; }
QPushButton:hover { background-color: rgba(230, 126, 34, 1.0); }</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableWidget" name="submissionTable">
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">QTableWidget { gridline-color: #bdc3c7; }
QHeaderView::section { background-color: #34495e; color: white; padding: 5px; font-weight: bold; }</string>
         </property>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <column>
          <property name="text">
           <string>ID</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Titre</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Auteur</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Domaine</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Date</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Statut</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Relecteur</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
